view.php

```diff
@@ -104,9 +104,9 @@
       . "<a href='$wwsetlink'>" . get_string("iframeNoShow-2", "wwassignment")
       ."</a><p align='center'></iframe></p>\n"
       );
-print("<iframe id='wwPage' src='$wwsetlink' frameborder='1' "
+print("<iframe id='wwPage' src='$wwsetlink' frameborder='0' "
       . "width='".$CFG->wwassignment_iframewidth."' "
-      . "height='".$CFG->wwassignment_iframeheight."'>"
+      . "height='".$CFG->wwassignment_iframeheight."' scrolling='no' >"
```

Comment:
I've done a quick review of the responsive capacities of putting everything without a scroll and everything seemed to fit. But I should be further tested...